lib/board.rb

```diff
@@ -0,0 +1,52 @@
+require 'pry'
+
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    self.reset!
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+    @cells
```

Comment:
The previous line, `@cells = Array.new(9, " ")` will already return `@cells` (the result of the assignment operation); you don't need to call `@cells` again separately.